syntax/grammar.mjs

```diff
@@ -295,37 +299,39 @@ let VariantExpression = defer(() =>
 let SelectExpression = defer(() =>
     sequence(
         InlineExpression.abstract,
-        maybe(inline_space),
+        maybe(blank_inline),
         string("->"),
+        maybe(blank_inline),
```

Comment:
Good catch, thanks. I changed it to `blank? "->" blank_inline?`. We still need `_inline` after the arrow because it's followed by a `Variant` which in turn must start with a `line_end`. `blank` would eagerly consume that `line_end`.